tests/test_generics.py

```diff
@@ -70,6 +70,60 @@ class TestModel(BaseXmlModel, tag='model1'):
     assert_xml_equal(actual_xml, xml)
 
 
+def test_generic_list():
+    GenericType = TypeVar('GenericType')
+
+    class GenericModel(BaseGenericXmlModel, Generic[GenericType], tag="model1"):
+        elems: list[GenericType] = element(tag="elem")
```

Comment:
`list[...]` syntax doesn't work in python < 3.9, lets change it to `List[GenericType]`

Reply:
Fixed in [cde29c2](https://github.com/dapper91/pydantic-xml/pull/66/commits/cde29c27f663bb5c48bae30e83079b504bb1f5be)